frontend/src/pages/RegisterStartup.tsx

```diff
@@ -0,0 +1,317 @@
+    if (!form.name) newErrors.name = "Не ввели прізвище";
+    if (!form.surname) newErrors.surname = "Не ввели ім’я";
+    if (!form.registercompany && !form.startup) newErrors.startup = "Виберіть кого ви представляєте";
+    if (!form.entrepreneur && !form.legal) newErrors.person = "Виберіть який суб’єкт господарювання ви представляєте";
+    setErrors(newErrors);
+    return Object.keys(newErrors).length === 0;
+  };
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+    if (!validate()) return;
+    setStatus("loading");
+    try {
+      await new Promise((resolve) => setTimeout(resolve, 1500));
+      setStatus("success");
+    } catch {
+      setStatus("error");
+    }
+  };
+
+  if (status === "success") {
+    return (
+      <div className="page">
+        <header className="header">
+          <div className="logo">MyCompany</div>
+          <nav>
+            <a href="#">О нас</a>
+            <a href="#">Предприятия</a>
+          </nav>
+            <div className="search-container">
+              <input type="text" placeholder="Поиск..."/>
+              <a href="#"><span className="search-icon">🔍</span></a>
+            </div>
+          <nav>
+            <a href="#">Войти</a>
+            <a href="#">Регистрация</a>
+          </nav>
+        </header>
+
+        <div className="container">
+          <div className="success-card">
+            <h2>Registration successful!</h2>
+            <p>Check your email to confirm your account.</p>
+          </div>
+        </div>
+
+        <footer className="footer"></footer>
+      </div>
+    );
+  }
+
+  return (
+      <div className="page">
+        <header className="navigation">
+          <div className="nav-container">
+            <div className="nav-logo">
+              <div className="logo-icon">
+                <img src={craftmergelogoblack} alt="CraftMerge logo"/>
+              </div>
+              <span className="logo-text">CraftMerge</span>
+            </div>
+
+            <nav className="nav-menu">
+              <div className="menu-item">
+                <span>Про нас</span>
+                <div className="underline"></div>
+              </div>
+              <div className="menu-item">
+                <span>Підприємства та сектори</span>
+                <div className="underline"></div>
+              </div>
+            </nav>
+
+            <div className="search-box">
+              <div className="wrapper">
+                <input className="search-input" type="text" placeholder="Пошук"/>
+              </div>
+              <div className="search-icon">
+                <svg width="16" height="16" viewBox="0 0 24 24" fill="none">
+                  <path
+                      d="M21 21L15 15M17 10C17 13.866 13.866 17 10 17C6.134 17 3 13.866 3 10C3 6.134 6.134 3 10 3C13.866 3 17 6.134 17 10Z"
+                      stroke="#25292C" stroke-width="2" stroke-linecap="round"/>
```

Comment:
JSX SVG attributes use HTML attribute names (stroke-width, stroke-linecap) which are invalid in React; use camelCase props instead. Replace them with strokeWidth and strokeLinecap in the SVG element.

```
<path
  d="M21 21L15 15M17 10C17 13.866 13.866 17 10 17C6.134 17 3 13.866 3 10C3 6.134 6.134 3 10 3C13.866 3 17 6.134 17 10Z"
  stroke="#25292C" strokeWidth={2} strokeLinecap="round" />
```